daemon/src/maker_cfd.rs

```diff
@@ -691,20 +691,12 @@ where
                     tracing::warn!("Failed to handle rollover proposal V2: {:#}", e);
                 }
             }
-            wire::TakerToMaker::RolloverProtocol { .. } => {
-                unreachable!("This kind of message should be sent to the rollover_maker::Actor`")
-            }
-            wire::TakerToMaker::Protocol { .. } => {
-                unreachable!("This kind of message should be sent to the `setup_maker::Actor`")
```

Comment:
nit: the previous error messages were better (at least for me), but I don't mind that much

Reply:
Same as my comment above - I don't think these different `unreachable` statements helped.

Reply:
as said above, don't mind too much with this one 🙃  it was just expressing an opinion about the contents of `unreachable` - it actually used to mention which actor handles it.